cmd20.py

```diff
@@ -1,13 +1,28 @@
 #!/usr/bin/env python3
 # Name: cmd20py
-# Authors: Luke Short | Joe R
 # Repo: https://github.com/ekultails/cmd20py/
 
+
+import secrets
+import string
 import random
 
-# Number Generator
-random.seed(None)
-die1 = random.randrange(0,7)
-die2 = random.randrange(0,7)
-die_results_msg = "You rolled a:"
-print(die_results_msg,die1,die2)
+def seed_generator():
+    """ Function for planting seeds """
+
+    ascii_char = 'string.ascii_letters + string.ascii_digits'
+    seed = ''.join(secrets.choice(ascii_char) for char in range(37))
+    return seed
+
+seed = seed_generator()
+
+def random_generator():
+    """ Number generator which uses seed_generator for extra randomness """
+
+    random.seed(seed)
```

Comment:
I believe random.SystemRandom() is a better bet for genuine random number generation.